### ISSUE TITLE
Printing problems

### ISSUE THREAD

**Issue by Cr4xy** [tomblind/local-lua-debugger-vscode#14]
Printing `{{}}` prints `}}`

This code causes the debugger to stop working/hang:
```
print("{method = \"test\"}")
print("{{}}")
```

**Comment by tomblind** [comment]
I can't seem to reproduce this. Can you show me your launch.json config?

**Comment by Cr4xy** [comment]
I think it might be because I'm using a logger (http://neopallium.github.io/lualogging/manual.html).
I'm seeing this printed in the console:
```
{
  "threadId": 32,
  "breakType": "breakpoint",
  "tag": "$luaDebug",
  "type": "debugBreak",
  "message": "breakpoint hit: \"...\""
}
```
Here's my launch.json:
```
{
  "name": "Local Lua Debugger",
  "type": "lua-local",
  "request": "launch",
  "cwd": "${workspaceFolder}",
  "program": {
    "lua": "lua5.1",
    "file": "${relativeFile}"
}
...

**Comment by tomblind** [comment]
In theory the logger shouldn't conflict with the debugger. I actually managed to reproduce this in one of my own projects the other day. As soon as a get some time, I'll try to debug and fix this.

**Comment by tomblind** [comment]
I've just published a new version (0.1.6) which I hope will fix this issue for you. Please let me know if it does (or doesn't).

**Comment by tomblind** [comment]
Closing this for now since as far as I know, it's fixed.
